src/modules/dashboard/components/Dialogs/DecisionDialog/DecisionDialog.tsx

```diff
@@ -50,27 +59,35 @@ const DecisionDialog = ({
         placeholder: 'Enter the description...',
       }),
     ],
-    content,
+    content: description,
   });
 
-  const handleSubmit = () => {
+  const domainId =
+    ethDomainId === COLONY_TOTAL_BALANCE_DOMAIN_ID || ethDomainId === undefined
+      ? ROOT_DOMAIN_ID
+      : ethDomainId;
+
+  const handleSubmit = (values) => {
     if (editor) {
       editor.setEditable(false);
-      // Do something with form contents
     }
+
+    localStorage.setItem(LOCAL_STORAGE_DECISION_KEY, JSON.stringify(values));
+    close();
+    history.push(`/colony/${colonyName}/decisions/preview`);
```

Comment:
Where is it said that go back should take you to ColonyHome. I think that it makes sense to go to Decisions page.
I am fine with either way but I think that it makes sense to go back to decisions page.
@arrenv , @danbr, where do you what the `Back` button to lead from the edit decision page: to Colony Decisions or to ColonyHome?

Reply:
@chinins I have not tested, based on Will's comment however, it seems like every time you click edit it adds a new history entry to the same preview page. It would be more desirable to have the back button go to the ColonyDecisions page after one click, independent from how many times you click the edit button.

Reply:
> @chinins I have not tested, based on Will's comment however, it seems like every time you click edit it adds a new history entry to the same preview page. It would be more desirable to have the back button go to the ColonyDecisions page after one click, independent from how many times you click the edit button.

Sure, I've added it.